### PR TITLE
Francesco add context [WIP]

### PR DESCRIPTION
**Pull request by Gamaranto**
This PR adds context and live data read from storage to the Proposals Tab. 

## What to Expect

The proposal tab should be "navigable" and fetch live data from the runtime. If a new proposal is submitted, the data should update with a page refresh. The CI build should pass before considering merging.

## What to Not Expect

There is no support sending extrinsics here, so voting, submitting proposals and the like won't work. (This will be addressed in a separate PR).

## Feedback

Any type of feedback is welcome 👀

### CODE REVIEW

File: packages/joy-types/src/index.ts [hunk 1]
```diff
@@ -167,62 +161,52 @@ function registerElectionAndProposalTypes() {
...
     typeRegistry.register({
-      'ElectionStake': {
-        'new': 'Balance',
-        'transferred': 'Balance'
-      },
-      'Backer': {
-        member: 'AccountId',
-        stake: 'Balance'
-      },
-      'Seat': {
```
Comment (Lezek123) [inline packages/joy-types/src/index.ts:178]: Some types seem to have been removed here, was it intentional?

File: packages/joy-types/src/proposals.ts [hunk 2]
```diff
@@ -47,101 +94,261 @@ export type Proposal = {
...
+export class Seats extends Vec.with(Seat) {}
 
 // export default proposalTypes;
 export function registerProposalTypes() {
   try {
     getTypeRegistry().register({
       ProposalId,
       ProposalStatus,
-      VoteKind
+      ProposalOf: Proposal,
```
Comment (Lezek123) [inline packages/joy-types/src/proposals.ts:342]: Does it mean we don't need "ProposalOf" struct/class defined above anymore?

File: packages/joy-proposals/src/Proposal/Loading.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+import { Loader, Container } from "semantic-ui-react";
+
+type LoadingProps = {
+  text: string;
+};
+
+export default function Loading({ text }: LoadingProps) {
+  return (
+    <Container style={{ display: "flex", justifyContent: "center", alignItems: "center" }}>
+      <Loader>{text}</Loader>
```
[the remark this answers is not in the document]
Comment (Gamaranto) [reply packages/joy-proposals/src/Proposal/Loading.tsx:11]: Fixed! Thanks!

File: packages/joy-types/src/proposals.ts (hunk 2 above)
Comment (Gamaranto) [reply packages/joy-types/src/proposals.ts:342]: Correct, will remove it

File: packages/joy-types/src/index.ts (hunk 1 above)
Comment (Gamaranto) [reply packages/joy-types/src/index.ts:178]: Yes, they have been moved to `packages/joy-types/src/proposals.ts`

**Pull request merged**